src/styles/transitions.spec.js

```diff
@@ -2,9 +2,25 @@
 /* eslint-env mocha */
 
 import { assert } from 'chai';
+import { spy } from 'sinon';
 import transitions, { easing, duration } from './transitions';
 
 describe('transitions', () => {
+  let originalConsoleError;
+
+  beforeEach(() => {
+    /* eslint-disable no-console */
+    originalConsoleError = console.error;
```

Comment:
i will probably remove them, it seems that flow server is not fond of using console.error variable and complains...